### PR TITLE
Editorial weekly 2023-04-20

### PR DESCRIPTION
**Pull request by pedro-hos**

### CODE REVIEW

File: src/content/posts/weekly-2023-04-20.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
+= This Week in JBoss - April, 06 2023
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:1]:
```suggestion
= This Week in JBoss - April 20, 2023
```

File: src/content/posts/weekly-2023-04-20.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Pedro Silva
+2023-04-20
+:tags: quarkus, java, wildfly, keycloak, byteman, kie-server
+
+Woww! "it's a me, another edition of the JBoss Editorial!" We are bringing exciting news and updates from your JBoss communities.
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:6]:
```suggestion
Wow! "It's another edition of the JBoss Editorial!" We are bringing exciting news and updates from your JBoss communities.
```

File: src/content/posts/weekly-2023-04-20.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* link:http://bytemanblog.blogspot.com/2023/04/byteman-4021-has-been-released.html[Byteman 4.0.21 has been released]
+* link:https://blog.kie.org/2023/04/kogito-1-36-0-released.html[Kogito 1.36.0 released!]
+* link:https://www.keycloak.org/2023/04/keycloak-2110-released[Keycloak 21.1.0 released]
+* https://quarkus.io/blog/vscode-quarkus-1.13.0-released/[New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0]
+
+=== How to configure WildFly with YAML files
+
+link:https://www.mastertheboss.com/jbossas/jboss-configuration/how-to-configure-wildfly-with-yaml-files/[How to configure WildFly with YAML files] by F.Marchioni
+
+WildFly 28 includes support for YAML configuration which is offers a more flexible approach in some use cases. In this tutorial we will discuss which are the best scenarios where YAML configuration is a perfect fit and how to configure WildFly to use YAML files.
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:23]:
```suggestion
WildFly 28 includes support for YAML configuration which offers a more flexible approach in some use cases. In this tutorial we will discuss which are the best scenarios where YAML configuration is a perfect fit and how to configure WildFly to use YAML files.
```

File: src/content/posts/weekly-2023-04-20.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+=== An Introduction To Server Connector for IntelliJ
+
+link:https://www.wildfly.org//news/2023/04/13/intellij-server-connector/[An Introduction To Server Connector for IntelliJ] by rstryker
+
+Server Connector for IntelliJ, also known as IntelliJ-rsp is a tool for acquiring, locating, installing, starting, stopping, and deloying artifacts to Application Servers like WildFly and EAP. On this article, the author shows to the lector on an introduction into the IntelliJ-rsp.
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:29]:
```suggestion
Server Connector for IntelliJ, also known as IntelliJ-rsp, is a tool for acquiring, locating, installing, starting, stopping, and deloying artifacts to Application Servers like WildFly and EAP. In this article, the author gives us an introduction into IntelliJ-rsp.
```

File: src/content/posts/weekly-2023-04-20.adoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+link:https://resteasy.dev/2023/04/19/tracing-feature-in-wildfly/[Using The RESTEasy Tracing Feature In WildFly] by Wei Nan Li
+
+The RESTEasy tracing feature has been integrated into WildFly since its version 28. In this article, Wei focuses on the usage of this feature in WildFly.
+
+=== How to Trace requests with RestEasy
+
+link:https://www.mastertheboss.com/jboss-frameworks/resteasy/how-to-trace-requests-with-resteasy/[How to Trace requests with RestEasy] by F.Marchioni
+
+In addition to the Wei Nan Li article's https://resteasy.dev/2023/04/19/tracing-feature-in-wildfly/[Using The RESTEasy Tracing Feature In WildFly] you can read the F. Marchioni text regarding the how to Trace requests with RestEasy
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:41]:
```suggestion
In addition to Wei Nan Li article's https://resteasy.dev/2023/04/19/tracing-feature-in-wildfly/[Using The RESTEasy Tracing Feature In WildFly], you can read F. Marchioni's text regarding the how to trace requests with RestEasy.
```

**Comment by pedro-hos** [comment]
@oraNod I've made the changes! Can you please review? Thank you

File: src/content/posts/weekly-2023-04-20.adoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+== Release roundup
+
+Here are the most recent releases for this edition:
+
+[square]
+
+* link:http://bytemanblog.blogspot.com/2023/04/byteman-4021-has-been-released.html[Byteman 4.0.21 has been released]
+* link:https://blog.kie.org/2023/04/kogito-1-36-0-released.html[Kogito 1.36.0 released!]
+* link:https://www.keycloak.org/2023/04/keycloak-2110-released[Keycloak 21.1.0 released]
+* https://quarkus.io/blog/vscode-quarkus-1.13.0-released/[New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0]
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:17]:
```suggestion
* link:https://quarkus.io/blog/vscode-quarkus-1.13.0-released/[New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0]
```

File: src/content/posts/weekly-2023-04-20.adoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
+= This Week in JBoss - April, 20 2023
+Pedro Silva
+2023-04-20
+:tags: quarkus, java, wildfly, keycloak, byteman, kie-server
+
+Wow! "it's another edition of the JBoss Editorial!" We are bringing exciting news and updates from your JBoss communities.
```
Comment (oraNod) [inline src/content/posts/weekly-2023-04-20.adoc:6]:
```suggestion
Wow! "It's another edition of the JBoss Editorial!" We are bringing exciting news and updates from your JBoss communities.
```

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks @pedro-hos

**Pull request merged**